nano/node/active_transactions.cpp

```diff
@@ -1000,17 +1003,18 @@ nano::inactive_cache_information nano::active_transactions::find_inactive_votes_
 
 void nano::active_transactions::erase_inactive_votes_cache (nano::block_hash const & hash_a)
 {
-	auto existing (inactive_votes_cache.get<tag_hash> ().find (hash_a));
-	if (existing != inactive_votes_cache.get<tag_hash> ().end ())
+	auto & inactive_by_hash (inactive_votes_cache.get<tag_hash> ());
+	auto existing (inactive_by_hash.find (hash_a));
+	if (existing != inactive_by_hash.end ())
 	{
-		inactive_votes_cache.get<tag_hash> ().erase (existing);
+		inactive_by_hash.erase (hash_a);
```

Comment:
Maybe keep `existing` here to avoid re-hashing, otherwise LGTM

Reply:
Ah yes